aiida_testing/export_cache/_fixtures.py

```diff
@@ -150,9 +157,10 @@ def _load_cache(path_to_cache=None, node=None, load_all=False):
         # for this we rehash all process nodes
         # this way we use the full caching mechanism of aiida-core.
         # currently this should only cache CalcJobNodes
-        qb = QueryBuilder()
-        qb.append(ProcessNode, tag='node')  # query for all ProcesNodes
-        to_hash = qb.all()
+        qub = QueryBuilder()
+        qub.append(ProcessNode)  # query for all ProcesNodes
+        qub.append(Code)
```

Comment:
This line breaks the cachability for a test in aiida-kkr: https://github.com/JuDFTteam/aiida-kkr/blob/bf474a2e37d4a54a0a38f8dbe96769847637c5bf/aiida_kkr/tests/calculations/test_vorocalc.py#L80

If I comment out `qub.append(Code)` it runs as expected.